crates/hyperswitch_domain_models/src/business_profile.rs

```diff
@@ -984,6 +1032,7 @@ impl super::behaviour::Conversion for BusinessProfile {
             tax_connector_id: self.tax_connector_id,
             is_tax_connector_enabled: Some(self.is_tax_connector_enabled),
             version: self.version,
+            dynamic_routing_algorithm: None,
```

Comment:
Why is this `None`?

Reply:
We will be deciding when to add it in v2, that is why made this None.